src/scripts/main.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const bodyElement = document.body;
+  const notificationElement = document.createElement('div');
+  const notificationTitleElement = document.createElement('h2');
+  const notificationDescriptionElement = document.createElement('p');
+
+  notificationElement.classList = `notification ${type}`;
+  notificationElement.style = `top: ${posTop}px; right: ${posRight}px;`;
```

Comment:
`style` is an object, so you can set values by keys: `element.style.top = '100px';`
It's a safer way, cause it doesn't overwrite other styles.